config/feed-variants/20220509-variant.json

```diff
@@ -0,0 +1,114 @@
+{
+  "max_days_since_published": 15,
+  "order_by": "relevancy_score_and_publication_date",
+  "levers": {
+    "daily_decay": {
+      "cases": [
+        [0, 1],
+        [1, 0.99],
+        [2, 0.985],
+        [3, 0.98],
+        [4, 0.975],
+        [5, 0.97],
+        [6, 0.965],
+        [7, 0.96],
+        [8, 0.955],
+        [9, 0.95],
+        [10, 0.945],
+        [11, 0.94],
+        [12, 0.935],
+        [13, 0.93],
+        [14, 0.925]
+      ],
+      "fallback": 0.9
+    },
+    "comments_count_by_those_followed": {
+      "cases": [
+        [0, 0.95],
+        [1, 0.98],
+        [2, 0.99]
+      ],
+      "fallback": 0.93
+    },
+    "comments_count": {
+      "cases": [
+        [0, 0.8],
+        [1, 0.82],
+        [2, 0.84],
+        [3, 0.86],
+        [4, 0.88],
+        [5, 0.9],
+        [6, 0.92],
+        [7, 0.94],
+        [8, 0.96],
+        [9, 0.98]
+      ],
+      "fallback": 1
+    },
+    "featured_article": { "cases": [[1, 1]], "fallback": 0.85 },
+    "following_author": {
+      "cases": [
+        [0, 0.8],
+        [1, 1]
+      ],
+      "fallback": 1
+    },
+    "following_org": {
+      "cases": [
+        [0, 0.95],
+        [1, 1]
+      ],
+      "fallback": 1
+    },
+    "latest_comment": {
+      "cases": [
+        [0, 1],
+        [1, 0.9988]
+      ],
+      "fallback": 0.988
+    },
+    "matching_negative_tags_intersection_count": {
+      "cases": [
+        [0, 1],
+        [1, 0.8],
+        [2, 0.5],
+        [3, 0.3],
+        [4, 0.2]
+      ],
+      "fallback": 0
+    },
+    "matching_positive_tags_intersection_count": {
+      "cases": [
+        [0, 0.7],
+        [1, 0.85],
+        [2, 0.93],
+        [3, 0.98],
+        [4, 1]
+      ],
+      "fallback": 1
+    },
+    "privileged_user_reaction_granular": {
+      "cases": [
+        [-2, 0.05],
+        [-1, 0.5],
+        [0, 0.9],
+        [1, 0.98],
+        [2, 1]
+      ],
+      "fallback": 0.9,
+      "very_negative_reaction_threshold": -10,
```

Comment:
The -10 is the threshold for one single negative reaction:

https://github.com/forem/forem/blob/main/app/models/reaction.rb#L2-L6

Reply:
Very negative. Such wow